pkgs/development/python-modules/torch/default.nix

```diff
@@ -117,8 +189,9 @@ let
         "8.7"
         "8.9"
         "9.0"
+        "9.0a"
       ];
-      ptx = lists.map (x: "${x}+PTX") real;
+      ptx = map (x: "${x}+PTX") real;
```

Comment:
Since we're reviewing the whole expression, this probably should be conditional on `cudaEnableForwardCompat`

Reply:
Good catch!

Reply:
Resolved in https://github.com/NixOS/nixpkgs/pull/319711/commits/269ed33e57ec379e822890ad1f9e879337045443